index.html

```diff
@@ -28,224 +28,247 @@ <h4>My Logo</h4>
+          <li class="social-icon">
+            <a href="https://angel.co/u/lilian-francis"> <img src="images/angellist.svg" alt="Lilian's AngelList" />
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="https://github.com/ifnotlily/">
+              <img src="images/github.svg" alt="Lilian's Github" />
+            </a>
+          </li>
+        </ul>
+      </div>
     </div>
-
   </section>
-  <section class="section-grid">
-    <div id="item-0">
-    </div>
-    <div id="item-1">
-      <h3>Tonic</h3>
+  <section class="nature-section">
+    <div class="natureimg">
+
     </div>
-    <div id="item-2">
-      <div class="flex-container">
-        <div class="flex-items">
-          <p>CANOPY &nbsp;<img src="/images/Counter.png" alt="canopy"></p>
-        </div>
-        <div class="flex-items">
-          <p>Back End Dev &nbsp;<img src="/images/Counter.png" alt="backend"></p>
+    <div class="section-grid">
+      <div id="item-0">
+      </div>
+      <div id="item-1">
+        <h3>Tonic</h3>
+      </div>
+      <div id="item-2">
+        <div class="flex-container">
+          <div class="flex-items">
+            <p>CANOPY &nbsp;<img src="/images/Counter.png" alt="canopy"></p>
+          </div>
+          <div class="flex-items">
+            <p>Back End Dev &nbsp;<img src="/images/Counter.png" alt="backend"></p>
+          </div>
+          <div class="flex-items">
+            <p>2015</p>&nbsp;
+          </div>
+          <div class="flex-items"></div>
         </div>
-        <div class="flex-items">
-          <p>2015</p>&nbsp;
+
+        <div id="item-3">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
         </div>
-        <div class="flex-items"></div>
+        <ul id="item-4">
+          <li class="button button1">html</li>
+
+          <li class="button button2">css</li>
+          <li class="button button3">javascript</li>
+        </ul>
+        <div id="item-7"><button class="buttonproject">See Project</button></div>
       </div>
     </div>
-    <div id="item-3">
-      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-    </div>
-    <ul id="item-4">
-      <li class="button button1">html</li>
-
-      <li class="button button2">css</li>
-      <li class="button button3">javascript</li>
-    </ul>
-    <div id="item-7"><button class="buttonproject">See Project</button></div>
-
   </section>
 
-  <section class="section-grid">
-    <div id="itemb-0">
-    </div>
-    <div id="itemb-1">
-      <h3>Multi-Post
-        Stories</h3>
-    </div>
-    <div id="itemb-2">
-      <div class="flex-container">
-        <div class="flex-items">
-          <p>CANOPY &nbsp;<img src="/images/Counter.png" alt="canopy"></p>
-        </div>
-        <div class="flex-items">
-          <p>Back End Dev &nbsp;<img src="/images/Counter.png" alt="backend"></p>
+
+  <section class="nature-section">
```

Comment:
Each one of the projects should have the image as requested by the previous reviewer 
###Original design
![right](https://user-images.githubusercontent.com/58376020/126868171-c81bea94-8aa5-4d51-a444-8d93c1f785de.png)
#### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/126868188-55bbd3da-c4b8-4750-a9da-8d3683925f79.png)